src/gui/editors/PianoRoll.cpp

```diff
@@ -4463,30 +4463,36 @@ bool PianoRoll::deleteSelectedNotes()
 void PianoRoll::autoScroll( const TimePos & t )
 {
 	const int w = width() - m_whiteKeyWidth;
-	if( t > m_currentPosition + w * TimePos::ticksPerBar() / m_ppb )
+	if (m_timeLine->autoScroll() == TimeLineWidget::AutoScrollState::Stepped) 
 	{
-		m_leftRightScroll->setValue( t.getBar() * TimePos::ticksPerBar() );
+		if(t > m_currentPosition + w * TimePos::ticksPerBar() / m_ppb)
```

Comment:
Sorry to bug you on style again, but we use spaces between if and the parantheses. I've seen similar style getting repeated. Correct those too.